presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -13,19 +13,27 @@ may be used to tune Presto or alter its behavior when required.
 General Properties
 ------------------
 
-``distributed-joins-enabled``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``join-distribution-type``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * **Type:** ``boolean``
-    * **Default value:** ``true``
-
-    Use hash distributed joins instead of broadcast joins. Distributed joins
-    require redistributing both tables using a hash of the join key. This can
-    be slower (sometimes substantially) than broadcast joins, but allows much
-    larger joins. Broadcast joins require that the tables on the right side of
-    the join after filtering fit in memory on each node, whereas distributed joins
-    only need to fit in distributed memory across all nodes. This can also be
-    specified on a per-query basis using the ``distributed_join`` session property.
+    * **Type:** ``string``
+    * **Allowed values:** ``AUTOMATIC``, ``REPARTITIONED``, ``REPLICATED``
```

Comment:
REPLICATED should be replaced with BROADCAST since that's the term that's always been used for presto, so it's what people are used to.